lib/expression/parser.go

```diff
@@ -126,6 +131,38 @@ func DefaultParserSpec[evaluationEnv any]() typical.ParserSpec[evaluationEnv] {
 				func(r remover, items ...string) (any, error) {
 					return r.remove(items...), nil
 				}),
+			"before": typical.BinaryFunction[evaluationEnv](
```

Comment:
Looks like this will add support for new functions in other contexts too. Don't forget to update the docs.